src/uct/tcp/tcp_ep.c

```diff
@@ -584,6 +615,7 @@ static inline ssize_t uct_tcp_ep_send(uct_tcp_ep_t *ep)
                                 &sent_length);
     if (ucs_unlikely((status != UCS_OK) &&
                      (status != UCS_ERR_NO_PROGRESS))) {
+        uct_tcp_ep_handle_send_err(ep, status);
```

Comment:
should we use the return value of uct_tcp_ep_handle_send_err()?

Reply:
done